pkg/model/initializers.go

```diff
@@ -104,7 +109,7 @@ ENTRY:
 	// First has more priority
 	priorityList := []string{
 		// First llama.cpp and llama-ggml
-		LLamaCPP, LlamaGGML, Gpt4All,
+		LLamaCPP, LlamaGGML, Gpt4All, LLamaCPPFallback,
```

Comment:
why fallback here? `LLamaCPP` should fallback automatically if runtime is not avx2 or avx

Reply:
right, but what if they fail when being started? e.g. missing shared libraries goes undetected until the grpc process is started

Reply:
I see, makes sense. Current GPU detection is pretty naive so that should definitely be improved by checking for the cuda libs.